site/content/blog/2023-07-05-zap-updates-june-2023/index.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "ZAP Updates - June 2023"
+summary: >
+  June 2023 updates and ongoing feature development statuses.
+images:
+- https://www.zaproxy.org/blog/2023-07-05-zap-updates-june-2023/images/zapbot-monthly-updates-social.png
+type: post
+tags:
+- blog
+- update
+date: "2023-07-05"
+authors: 
+- akshath
+---
+
+Welcome to our monthly updates blog post!  
+There were a lot of incremental updates to ZAP in June 2023.
+
+## Highlights
+### Browser-Based Authentication Support in the AJAX Spider
+The AJAX Spider now supports crawling authenticated web pages with the browser-based authentication method.
+ZAP now runs regularly scheduled scans against OWASP Juice Shop that uses this mode of authentication. You can view more details about the scan [here](/docs/scans/juiceshop/).
+
+### Selenium 4
+All ZAP add-ons are now using [Selenium](https://www.selenium.dev/) v4.
```

Comment:
Add something like: One significant improvement this brings is that messages written to stdout by browsers will no longer be shown in the ZAP logs.